cypress/utils/Util.ts

```diff
@@ -0,0 +1,114 @@
+export function uniqueName(prefix: string) {
+  if (isEnvironmentType(EnvType.Mocked)) {
+    return prefix;
+  } else if (isEnvironmentType(EnvType.Dev)) {
+    return `testui-${prefix}-${Cypress._.uniqueId(Date.now().toString())}`;
+  } else {
+    throw new Error(
+      "The environment type is not recognized. Values of CYPRESS_TEST_TYPE are defined by the 'EnvType' enum."
+    );
+  }
+}
+
+export function pageWasLoaded() {
+  cy.get("#nav-toggle", { timeout: 30000 }).should("be.visible");
```

Comment:
I have added wait on some basic element to be sure that page is loaded correctly.
Otherwise next waiting rutine (loading-table) was succesful even if the page was empty.